### PR TITLE
Refactor: Chinese Remainder Theorem in Python

### PR DESCRIPTION
**Pull request by charlie219**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

As assigned by @shraddhavp for issue #1610, I have carefully restructured the Python `README.md ` file. All the Algorithms inside the Python folder are distinctly mentioned. 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1610 

I request @shraddhavp @iamrajiv @HarshCasper to review this PR. Thank You 👍

### CODE REVIEW

**Review by iamrajiv — APPROVED** [review]

File: Python/README.md [hunk 1]
```diff
@@ -110,66 +44,69 @@
...
-* [Large Number Factorial](math/Large_number_factorial.py)
-* [Sieve of Eratosthenes](math/prime_number_using_sieve_of_Eratosthenes.py)
-* [Fibonacci number](math/Fibonacci.py)
-* [Strassen's Matrix Multiplication](math/Strassen's_Matrix_Mul.py)
-* [Check Armstrong Number](math/CheckArmstrongNumber.py)
-- [Prim's Algorithm](graphs/Prims_Algo_dynamic.py)
-- [Kruskal's Algorithm](graphs/Kruskal_Algorithm.py) 
+- [Floyd Warshall Algorithm](graphs/Floyd_Warshall_Algorithm.py)
+- [Knight Walk](graphs/Knight_Walk.py)
+- [Bridge Edge in Graph](graph/Bridge_Edge_in_Graph.py)
```
Comment (atarax665) [inline Python/README.md]:
```suggestion
- [Bridge Edge in Graph](graphs/Bridge_Edge_in_Graph.py)
```

File: Python/README.md [hunk 2]
```diff
@@ -178,98 +115,78 @@
...
 - [Kadane's Algorithm](other/Kadanes_Algorithm.py)
-- [Least Common multiple](other/Least_common_multiple.py)
 - [Longest Substring](other/longest_Substring.py)
 - [Merging Sorted arrays](other/merge_sorted_arrays.py)
 - [Maximum Square Matrix with all 1's](other/Maximum_Square_Matrix.py)
 - [Possible Code Recursion](other/Possible_code_recursion.py)
 - [Reverse a string using stack](other/Reverse_string_using_stack.py)
 - [Russian Peasant Algorithm](other/Russian_Peasant_Algorithm.py)
+- [Ocurrence of a number using binary search](other/find_freq_using_bs.py)
+- [Chinese Remainder theoram](other/chinese_remainder_theorem.py)
```
Comment (atarax665) [inline Python/README.md]:
```suggestion
- [Chinese Remainder theorem](other/chinese_remainder_theorem.py)
```

**Review by atarax665 — CHANGES_REQUESTED** [review]
I think Chinese Remainder Theorem should be in ```math``` folder. Do change its location from ```other``` folder to ```math``` folder. You can change the file location if you find it to be in a wrong folder and make the required changes in README.md, I will review it once before merging.

File: Python/README.md (hunk 1 above)
Comment (charlie219) [reply Python/README.md]: i have made the required changes

File: Python/README.md (hunk 2 above)
Comment (charlie219) [reply Python/README.md]: Okay I will change the location of the Chinese Remainder Theorem to maths folder and add the description to it as i use to do in all my PRs.

**Comment by charlie219** [comment]
@atarax665 as you requested, I have relocated the Chinese Remainder Theorem form others to math folder. In doing so I have also done the following addition:
- Added description of the Chinese Remainder Theorem and the Problem Statement
- Function names modified to specify their usage and comments added
- Time/Space Complexity  included

I hope you appreciate the changes made by me. Thank You 👍

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Pull request merged**